tests/roles/run_tests/vars/main.yml

```diff
@@ -39,6 +39,7 @@ BAREMETALV6_POOL_RANGE_START: "{{ lookup('env', 'BAREMETALV6_POOL_RANGE_START')
 BAREMETALV6_POOL_RANGE_END: "{{ lookup('env', 'BAREMETALV6_POOL_RANGE_END') | default('fd55::200', true) }}"
 EXTERNAL_SUBNET_V6_PREFIX: "{{ lookup('env', 'EXTERNAL_SUBNET_V6_PREFIX') | default('64', true) }}"
 EXTERNAL_SUBNET_V6_HOST: "{{ lookup('env', 'EXTERNAL_SUBNET_V6_HOST') | default('fd55::1', true) }}"
+PROVISIONING_IP: "{{ lookup('env', 'PROVISIONING_IP') | default('172.22.0.1', true) }}"
```

Comment:
Do we need to add it in lib/common.sh as other vars or no need?

Reply:
I would say no. It is in lib/network.sh and that should be enough. I just added it here in the ansible vars now so it will also respect it if we set a different value. Previously we had a hard coded value in ansible.

Reply:
ah missed that, cool, thats enough